packages/viewer/src/components/viewer-intersection-query-tool/viewer-intersection-query-tool.tsx

```diff
@@ -0,0 +1,170 @@
+import { Component, Element, h, Host, Prop, State, Watch } from '@stencil/core';
+import { Disposable } from '@vertexvis/utils';
+
+import { VolumeIntersectionQueryController } from '../../lib/volume-intersection/controller';
+import { VolumeIntersectionQueryInteractionHandler } from '../../lib/volume-intersection/interactions';
+import {
+  QueryType,
+  VolumeIntersectionQueryDetails,
+  VolumeIntersectionQueryModel,
+} from '../../lib/volume-intersection/model';
+
+export type DefaultOperationType = 'select' | 'deselect';
+
+@Component({
+  tag: 'vertex-viewer-intersection-query-tool',
+  styleUrl: 'viewer-intersection-query-tool.css',
+  shadow: true,
+})
+export class ViewerIntersectionQueryTool {
+  /**
+   * The viewer that this component is bound to. This is automatically assigned
+   * if added to the light-dom of a parent viewer element.
+   */
+  @Prop()
+  public viewer?: HTMLVertexViewerElement;
+
+  /**
+   * The controller that is responsible for performing operations using the
+   * intersection query and updating the model.
+   */
+  @Prop({ mutable: true })
+  public controller?: VolumeIntersectionQueryController;
+
+  /**
+   * The model that contains the points representing the corners of the box
+   * displayed on screen, the type of the query to be performed, and methods
+   * for setting these values.
+   */
+  @Prop({ mutable: true })
+  public model?: VolumeIntersectionQueryModel;
+
+  /**
+   * The default operation to perform when a drag has completed and the intersection
+   * query will be run. Defaults to `select`, and can be changed to `deselect`.
+   *
+   * The operation behavior for this intersection query tool can also be changed by
+   * providing a custom implementation of the `VolumeIntersectionQueryController`, or
+   * by using the `setOperationTransform` method of the default controller.
+   */
+  @Prop()
+  public defaultOperation: DefaultOperationType = 'select';
```

Comment:
Should this just be named, `type` or `operationType`?

Reply:
I'd probably lean toward `operationType`, but kept the `default` to indicate that it can be overridden in the controller. Granted with the comment indicating that's possible, that may be redundant - I'll update this